control-plane/rest/service/src/v0/block_devices.rs

```diff
@@ -0,0 +1,36 @@
+use super::*;
+use mbus_api::v0::ListBlockDevices;
+
+pub(super) fn configure(cfg: &mut paperclip::actix::web::ServiceConfig) {
+    cfg.service(list_block_devices);
+}
+
+// List block devices takes a query parameter 'all' which is used to determine
+// whether to return all found devices or only those that are usable.
+// Omitting the query parameter will result in all block devices being shown.
+//
+// # Examples
+// List only usable block devices with query parameter:
+//      curl -X GET "https://localhost:8080/v0/nodes/mayastor/block_devices/list?all=false" \
+//      -H  "accept: application/json"
+//
+// List all block devices with query parameter:
+//      curl -X GET "https://localhost:8080/v0/nodes/mayastor/block_devices/list?all=true" \
+//      -H  "accept: application/json" -k
+//
+// List all block devices without query parameter:
+//      curl -X GET "https://localhost:8080/v0/nodes/mayastor/block_devices/list" \
+//      -H  "accept: application/json" -k
+//
+#[get("/v0", "/nodes/{node}/block_devices/list", tags(BlockDevices))]
```

Comment:
the list/get is implicit, so should be just `/nodes/{node}/block_devices`

Reply:
Good point. Dropped the `/list` from the end of the uri.